pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -457,6 +457,9 @@ func Convert_v1_Input_To_api_InputDevice(input *v1.Input, inputDevice *Input, _
 	}
 
 	inputDevice.Bus = input.Bus
+	if inputDevice.Bus == "" {
+		inputDevice.Bus = "virtio"
+	}
```

Comment:
there is 
```
if input.Bus != "virtio" && input.Bus != "usb" {
	input.Bus = "usb"
}
```
just before this, so I guess it'll always be `usb` if it wasn't set to `virtio` here

Reply:
yes that is interesting ...

Reply:
oh man, I added that code. I thought it was there before. Thanks @vladikr  :+1:

Reply:
fixed.